lib/journald/src/lib.rs

```diff
@@ -0,0 +1,113 @@
+#![allow(non_camel_case_types)]
```

Comment:
minor nit: can we move this allow to the specific places it needs to be applied?

Reply:
Will do